### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by Ayaphis**

### CODE REVIEW

File: assets/data/doYouKnow.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-  "概览面板舰队右边的小箭头可以打开详细信息",
-  "设置里面可以字体的大小",
-  "设置面板里面可以设置关闭时提示，不必担心误关闭了",
-  "有些插件不自带 可以到github上下载",
-  "如果游戏有规律地无限卡死，很可能需要删掉自定义缓存",
-  "poi自带的插件可以减少查wiki的次数喔，比如 远征信息 和 任务信息",
-  "poi绝不会修改发包，所以如果猫了的话真的不是poi的问题poi",
-  "活动图的海域血量，血条会在下次选择出击地图时更新",
-  "自动适慢的可能是岛风go用户喔"
+  "In the prophet plugin, you can choose to show or hide the level information with mouse double click",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: [Prophet plugin] Hide/show ship levels by double click.
or 
*Toggle the display of ship levels*

File: assets/data/doYouKnow.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+  "Moving the cursor on the Admiral's Level, the NEXT experience will be shown",
+  "When resources are under the limited score, icon of resource will glisten",
+  "You can use the enter key in address field now",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: The address bar in Settings now supports Enter key.

File: assets/data/doYouKnow.json [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+  "If poi display the Couldn’t load plugin, maybe the reason is that your catalog is too long",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: The error "Couldn't load plugin" probably means the path to poi is too long.

File: assets/data/doYouKnow.json [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
+  "The boss key of poi is Ctrl/Cmd+i",
+  "The task あ号 was calculated by local counter of poi, so you must use same client of poi",
+  "Ctrl/Cmd+1 can switch to overview, Ctrl/Cmd+2 can switch to fleet info, Ctrl/Cmd+3 can swith to plugins, F5 can refresh web",
+  "The small arrow near the overview panel can open the detail of fleet info",
+  "You can change the font size in setting",
+  "You can open the prompt in the setting panel, it can help you to avoid turned poi off by accident",
+  "You can download other plugins in GitHub",
+  "If you always dumped in game play, maybe you need to try delete cache and cookies",
+  "Using plugins of poi, such as ship girls info and quest info, the frequency of wiki checking will be obsievely reduced",
+  "poi will not revise the data packets, so if you caught in a CatBomb(network error), it is not made by poi",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: poi never modifies network packets, and is not responsible for Cat Bombs.
or
*so please don't blame poi for Cat Bombs.*

File: assets/data/doYouKnow.json [hunk 5]
```diff
@@ -1,21 +1,21 @@
...
+  "Map HP will update when you chooseing sortie map",
+  "If the built-in browser is adjusted slowly, you must using ShimakazeGO"
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: Slow auto-adjustment is probably caused by the proxy, such as ShimakazeGo.
真的会有英语用户需要这一条么……
以及可能需要在auto-adjustment前加个限定语……但不知道用什么。我觉得用browser有点奇怪，觉得最像的是webview，不过这好像并不是一个英语词汇……

File: assets/data/doYouKnow.json [hunk 6]
```diff
@@ -1,21 +1,21 @@
...
+  "The prophet plugin doesn’t calculate opening aerial combat",
+  "The boss key of poi is Ctrl/Cmd+i",
+  "The task あ号 was calculated by local counter of poi, so you must use same client of poi",
+  "Ctrl/Cmd+1 can switch to overview, Ctrl/Cmd+2 can switch to fleet info, Ctrl/Cmd+3 can swith to plugins, F5 can refresh web",
+  "The small arrow near the overview panel can open the detail of fleet info",
+  "You can change the font size in setting",
+  "You can open the prompt in the setting panel, it can help you to avoid turned poi off by accident",
+  "You can download other plugins in GitHub",
+  "If you always dumped in game play, maybe you need to try delete cache and cookies",
+  "Using plugins of poi, such as ship girls info and quest info, the frequency of wiki checking will be obsievely reduced",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: Tired of looking up the wiki? Try our plugins for expedition and quests!

File: assets/data/doYouKnow.json [hunk 7]
```diff
@@ -1,21 +1,21 @@
...
-  "设置面板里面可以设置关闭时提示，不必担心误关闭了",
-  "有些插件不自带 可以到github上下载",
-  "如果游戏有规律地无限卡死，很可能需要删掉自定义缓存",
-  "poi自带的插件可以减少查wiki的次数喔，比如 远征信息 和 任务信息",
-  "poi绝不会修改发包，所以如果猫了的话真的不是poi的问题poi",
-  "活动图的海域血量，血条会在下次选择出击地图时更新",
-  "自动适慢的可能是岛风go用户喔"
+  "In the prophet plugin, you can choose to show or hide the level information with mouse double click",
+  "Moving the cursor on the Admiral's Level, the NEXT experience will be shown",
+  "When resources are under the limited score, icon of resource will glisten",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: A resource icon glows if that resource is regenerating.

File: assets/data/doYouKnow.json [hunk 8]
```diff
@@ -1,21 +1,21 @@
...
+  "If poi display the Couldn’t load plugin, maybe the reason is that your catalog is too long",
+  "Moving the cursor to the name of ship panel can check information about LOS and so on",
+  "The prophet plugin doesn’t calculate opening aerial combat",
+  "The boss key of poi is Ctrl/Cmd+i",
+  "The task あ号 was calculated by local counter of poi, so you must use same client of poi",
+  "Ctrl/Cmd+1 can switch to overview, Ctrl/Cmd+2 can switch to fleet info, Ctrl/Cmd+3 can swith to plugins, F5 can refresh web",
+  "The small arrow near the overview panel can open the detail of fleet info",
+  "You can change the font size in setting",
+  "You can open the prompt in the setting panel, it can help you to avoid turned poi off by accident",
+  "You can download other plugins in GitHub",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: Some downloadable plugins are available on Github.

File: assets/data/doYouKnow.json [hunk 9]
```diff
@@ -1,21 +1,21 @@
...
+  "If you always dumped in game play, maybe you need to try delete cache and cookies",
+  "Using plugins of poi, such as ship girls info and quest info, the frequency of wiki checking will be obsievely reduced",
+  "poi will not revise the data packets, so if you caught in a CatBomb(network error), it is not made by poi",
+  "Map HP will update when you chooseing sortie map",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: [Map HP plugin] The gauges are updated by entering the map choosing screen.

File: assets/data/doYouKnow.json [hunk 10]
```diff
@@ -1,21 +1,21 @@
...
-  "poi自带的插件可以减少查wiki的次数喔，比如 远征信息 和 任务信息",
-  "poi绝不会修改发包，所以如果猫了的话真的不是poi的问题poi",
-  "活动图的海域血量，血条会在下次选择出击地图时更新",
-  "自动适慢的可能是岛风go用户喔"
+  "In the prophet plugin, you can choose to show or hide the level information with mouse double click",
+  "Moving the cursor on the Admiral's Level, the NEXT experience will be shown",
+  "When resources are under the limited score, icon of resource will glisten",
+  "You can use the enter key in address field now",
+  "If poi display the Couldn’t load plugin, maybe the reason is that your catalog is too long",
+  "Moving the cursor to the name of ship panel can check information about LOS and so on",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: Hover the mouse over a fleet name for detailed information.

File: assets/data/doYouKnow.json [hunk 11]
```diff
@@ -1,21 +1,21 @@
...
-  "设置里面可以字体的大小",
-  "设置面板里面可以设置关闭时提示，不必担心误关闭了",
-  "有些插件不自带 可以到github上下载",
-  "如果游戏有规律地无限卡死，很可能需要删掉自定义缓存",
-  "poi自带的插件可以减少查wiki的次数喔，比如 远征信息 和 任务信息",
-  "poi绝不会修改发包，所以如果猫了的话真的不是poi的问题poi",
-  "活动图的海域血量，血条会在下次选择出击地图时更新",
-  "自动适慢的可能是岛风go用户喔"
+  "In the prophet plugin, you can choose to show or hide the level information with mouse double click",
+  "Moving the cursor on the Admiral's Level, the NEXT experience will be shown",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: Hover the mouse over HQ level for the XP needed to level up.

**Comment by Ayaphis** [comment]
感謝します，ＤＫちゃん

File: assets/data/doYouKnow.json [hunk 12]
```diff
@@ -1,21 +1,21 @@
...
+  "Moving the cursor to the name of ship panel can check information about LOS and so on",
+  "The prophet plugin doesn’t calculate opening aerial combat",
+  "The boss key of poi is Ctrl/Cmd+i",
+  "The task あ号 was calculated by local counter of poi, so you must use same client of poi",
+  "Ctrl/Cmd+1 can switch to overview, Ctrl/Cmd+2 can switch to fleet info, Ctrl/Cmd+3 can swith to plugins, F5 can refresh web",
+  "The small arrow near the overview panel can open the detail of fleet info",
+  "You can change the font size in setting",
+  "You can open the prompt in the setting panel, it can help you to avoid turned poi off by accident",
+  "You can download other plugins in GitHub",
+  "If you always dumped in game play, maybe you need to try delete cache and cookies",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: Try to clear the cache if you get stuck regularly.
* (especially after an update)   <- 我觉得可以加上这句？

File: assets/data/doYouKnow.json [hunk 13]
```diff
@@ -1,21 +1,21 @@
...
+  "You can use the enter key in address field now",
+  "If poi display the Couldn’t load plugin, maybe the reason is that your catalog is too long",
+  "Moving the cursor to the name of ship panel can check information about LOS and so on",
+  "The prophet plugin doesn’t calculate opening aerial combat",
+  "The boss key of poi is Ctrl/Cmd+i",
+  "The task あ号 was calculated by local counter of poi, so you must use same client of poi",
+  "Ctrl/Cmd+1 can switch to overview, Ctrl/Cmd+2 can switch to fleet info, Ctrl/Cmd+3 can swith to plugins, F5 can refresh web",
+  "The small arrow near the overview panel can open the detail of fleet info",
+  "You can change the font size in setting",
+  "You can open the prompt in the setting panel, it can help you to avoid turned poi off by accident",
```
Comment (dkwingsmt) [inline assets/data/doYouKnow.json]: You can enable a confirmation when closing the window in Settings.

File: assets/data/doYouKnow.json (hunk 4 above)
Comment (Coffee98) [inline assets/data/doYouKnow.json]: blame会不会不太好...
Cat Bombs(network error) is not caused by poi, and poi never modifies network packets.
So poi has no obligation to bear the mistakes of Cat Bombs?

**Comment by Ayaphis** [comment]
obsolete

**Pull request closed without merging**